binplace.targets

```diff
@@ -35,6 +35,9 @@
       <TargetingPackDir Include="$(RefPath)" />
       <TargetingPackDir Condition="'$(IsNETCoreAppRef)' == 'true'" Include="$(BinDir)netcoreapp\pkg\ref" />
     </ItemGroup>
+    <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('UAP'))">
```

Comment:
I think that's broken.  Consider I build a project with only a netstandard configuration for both BuildConfiguration netcoreapp and uap: separate vertical builds on same machine.  It will reuse the same intermediate/bin directories (per design) but with different refs we'll potentially have different assemblies.